community/faq.md

```diff
@@ -14,7 +14,7 @@ All files in the `enterprise/` and `src/enterprise/` directories are subject to
 
 ## Is all of Sourcegraph open source?
 
-All files in this repository except those in the `enterprise/` and `src/enterprise/` directories are open source, and build a product known as Sourcegraph OSS. Sourcegraph OSS omits certain trademarks, logos, and [paid enterprise features](https://about.sourcegraph.com/pricing/) from the official Sourcegraph build to make it open source.
+All files in this repository except those in the `enterprise/` and `client/web/src/enterprise/` directories are open source, and build a product known as Sourcegraph OSS. Sourcegraph OSS omits certain trademarks, logos, and [paid enterprise features](https://about.sourcegraph.com/pricing/) from the official Sourcegraph build to make it open source.
```

Comment:
Thanks for updating this page! Can we link to the new licensing.md page from this FAQ question somewhere? And perhaps to https://about.sourcegraph.com/handbook/ce/enterprise-vs-oss as well? We have a lot of distributed info on OSS across the handbook and I think we could improve usability with more links between them.